presto-main/src/main/java/com/facebook/presto/execution/LocalLoggingWarningSink.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.execution;
+
+import com.facebook.presto.spi.WarningCode;
+import com.google.common.collect.ImmutableList;
+import io.airlift.log.Logger;
+
+import java.util.Collection;
+
+import static java.util.Objects.requireNonNull;
+
+public class LocalLoggingWarningSink
+        implements WarningSink
+{
+    private static final Logger log = Logger.get(WarningSink.class);
+
+    public void logWarning(WarningCode warning, String message)
+    {
+        log.warn(new PrestoWarning(requireNonNull(warning, "warning is null"), requireNonNull(message, "message is null")).toString());
```

Comment:
move the `requireNonNull` to separate lines. It's hard to read the way it's currently written.